src/puppetlabs/puppetdb/dashboard.clj

```diff
@@ -28,3 +28,16 @@
                 (compojure/context (get-route this) [])
                 (add-ring-handler this))
            context)))
+
+(def dashboard-redirect
+  (app ["" &] {:get (fn [req] (rr/redirect "/pdb"))}))
```

Comment:
This should probably be `/pdb/dashboard/index.html` since `/pdb` will yet again redirect you